09-hypothesis-testing.Rmd

```diff
@@ -1338,28 +1346,30 @@ all_points <- tibble(
   `t: df = 3` = dt(x = domain, df = 3),
   `t: df = 10` = dt(x = domain, df = 10),
   `z` = dnorm(x = domain)
-)  %>% 
-  gather(key = "Distribution", value = "value", - domain) %>% 
+) %>%
+  gather(key = "Distribution", value = "value", -domain) %>%
   mutate(
     type = ifelse(Distribution == "z", "z", "t"),
     Distribution = factor(
-      Distribution, 
+      Distribution,
       levels = c("z", "t: df = 10", "t: df = 3", "t: df = 1")
-      )
     )
+  )
 
-for_labels <- all_points %>% 
+for_labels <- all_points %>%
   filter(between(domain, -1.205, -1.195) & Distribution == "t: df = 1" |
```

Comment:
Maybe it's a little cleaner to have `filter(` be on its own line here and then have the `)` after `Distribution == "z"` be on its own line too?